UPDATING.md

```diff
@@ -0,0 +1,54 @@
+# Updating models
+
+A convenience script has been provided, `./update-models.sh`.
+
+The process that's been automated here is as follows:
+
+```bash
+# install and get the generator into place
+git clone https://github.com/openactive/models-lib
+cd models-lib
+npm install
+
+# generate models
+npm run app-dev -- generate PHP --destination ../src/
```

Comment:
Should this be something like:
```diff
npm run app-dev -- generate PHP --destination ../models-php/src
cd ../models-php
```

Assuming you clone the generator in one folder up from `models-php`, and then run the `composer fix` command in `models-php`.